Pluto

```diff
@@ -5,3 +5,5 @@ P.S. 2,047 internet points for this reference.
 P.P.S You get 2^2,203 − 1 points for the 2047 reference. 
 
 @neiltyson is a hero.
+
+Pluto is the coolest and still totally a planet.
```

Comment:
I think it's really interesting that there seems to be such an age-divide between people who have embraced Pluto's demotion from planetary status (often older than 30) and those who staunchly oppose it (younger than 30).  On a related note, what's your opinion of Ewoks?